index.yaml

```diff
@@ -663,3 +652,47 @@ indexes:
   - name: is_root_pipeline
   - name: start_time
     direction: desc
+
```

Comment:
Hmm, interesting how many things are added here. Can you please check whether all these changes are expected?

Reply:
Am, after checking of them individually, I found that each of them is directly or indirectly related to the exploration related fields. Still, I feel like some of them are unnecessary. This file is generated automatically, right? So, if I delete the newly added line and perform reverting of an exploration on local dev server then lines which are absolutely necessary should be added again. Should I try this ?

Reply:
Yes, it should work that way, please try it and see what happens.

Reply:
Tried it, surprisingly nothing added to `index.yaml`.